changes/1874.feature.rst

```diff
@@ -0,0 +1 @@
+Added support for maximize, minimize, full_screen in WinForms.
```

Comment:
```suggestion
An API for maximising and minimising windows was added.
```

Also - this should be tied to the ticket discussing the feature: #1857 - rather than the PR.